src/QMCWaveFunctions/Jastrow/RPAJastrow.cpp

```diff
@@ -185,7 +207,7 @@ void RPAJastrow::makeShortRange()
   Y[npts]              = 0.0;
   std::string functype = "rpa";
   std::string useit    = "no";
-  nfunc->initialize(npts, X, Y, SRA->df(0), Rcut, functype, useit);
+  nfunc->initialize(nparam, X, Y, cusp, Rcut+tiny, functype, useit);
```

Comment:
A comment on why "tiny" is being added to Rcut would be nice.  This seems to be the result of a "gotcha"?

Reply:
"tiny" is there because of numerical issue. If I set Rcut to L/2 exactly, then BsplineFunctor may complain.

The old code had `Rcut-0.1`, which is strictly worse IMO. I think "tiny" should also be an input parameter that defaults to a very small number.

Reply:
What BsplineFunctor complained?

Reply:
```
ERROR ERROR: Error in BsplineFunctor::initialize: r > cutoff_radius.
Fatal Error. Aborting at BsplineFunctor::initialize
```